File: cpp/Program/Product.cpp
#include <iostream>
#include <string>

using namespace std;

class Product
{
    private:
        string id;
        string name;
        string brand;
        int price;

    public:
        Product(){
        }
        
        void setId(string id){
            this->id = id;
        }

        string getId(){
            return this->id;
        }

        void setName(string name){
            this->name = name;
        }

        string getName(){
            return this->name;
        }

        void setBrand(string brand){
            this->brand = brand;
        }

        string getBrand(){
            return this->brand;
        }

        void setPrice(int price){
            this->price = price;
        }

        int getPrice(){
            return this->price;
        }

        ~Product(){
        }
};

File: python/Program/Product.py
class Product:
    def __init__(self):
        self.__id = ""
        self.__name = ""
        self.__brand = ""
        self.__price = 0

    def set_id(self, id):
        self.__id = id

    def get_id(self):
        return self.__id

    def set_name(self, name):
        self.__name = name

    def get_name(self):
        return self.__name

    def set_brand(self, brand):
        self.__brand = brand

    def get_brand(self):
        return self.__brand

    def set_price(self, price):
        self.__price = price

    def get_price(self):
        return self.__price